gammapy/scripts/image_pipe.py

```diff
@@ -2,56 +2,160 @@
+        events = events.select_offset(self.offset_band)
+        self.maps["counts"].fill(events=events)
+        self.maps["counts"].data = self.maps["counts"].data.value
         log.info('Making counts image ...')
 
-    def make_exposure(self):
-        log.info('Making exposure image ...')
+    def make_total_counts(self):
+        """Stack the total count from the observation in the 'DataStore'
+
+        """
+        self.maps["total_counts"] = SkyMap.empty_like(self.empty_image)
+        for obs_id in self.obs_table['OBS_ID']:
+            self.make_counts(obs_id)
+            self.maps["total_counts"].data += self.maps["counts"].data
+
+    def make_background(self, obs_id):
+        """Make the background map for one observation from a bkg model
+
+        Parameters
+        ----------
+        obs_id : int
+            Number of the observation
+
+        """
+        self.maps["bkg"] = SkyMap.empty_like(self.empty_image)
+        obs = self.data_store.obs(obs_id=obs_id)
+        table = obs.bkg.acceptance_curve_in_energy_band(energy_band=self.energy_band)
+        center = obs.pointing_radec.galactic
+        bkg_hdu = fill_acceptance_image(self.header, center, table["offset"], table["Acceptance"], self.offset_band[1])
+        livetime = obs.observation_live_time_duration
+        self.maps["bkg"].data = Quantity(bkg_hdu.data, table["Acceptance"].unit) * self.maps[
+            "bkg"].solid_angle() * livetime
+        self.maps["bkg"].data = self.maps["bkg"].data.decompose()
+        self.maps["bkg"].data = self.maps["bkg"].data.value
 
-    def make_background(self):
         log.info('Making background image ...')
 
+    def make_background_normalized_offcounts(self, obs_id):
+        """Normalized the background compare to te events in the counts maps outside the exclusion maps
+
+        Parameters
+        ----------
+        obs_id : int
+            Number of the observation
+
+        """
+        self.make_counts(obs_id)
+        self.make_background(obs_id)
+        counts_sum = np.sum(self.maps["counts"].data * self.maps['exclusion'].data)
+        bkg_sum = np.sum(self.maps["bkg"].data * self.maps['exclusion'].data)
+        scale = counts_sum / bkg_sum
+        self.maps["bkg"].data = scale * self.maps["bkg"].data
+
+    def make_total_bkg(self):
+        """Stack the total bkg from the observation in the 'DataStore'.
+        """
+        self.maps["total_bkg"] = SkyMap.empty_like(self.empty_image)
+        for obs_id in self.obs_table['OBS_ID']:
+            self.make_background_normalized_offcounts(obs_id)
+            self.maps["total_bkg"].data += self.maps["bkg"].data
+
+    def make_significance(self, radius):
+        """Make the significance image from the counts and kbg images.
+
+        Parameters
+        ----------
+        radius : float
+            Disk radius in pixels.
+
+        """
+        self.maps["significance"] = SkyMap.empty_like(self.empty_image)
```

Comment:
Same comment as above: compute first, store as data member at the end.
Move log.info statement on the first line and change to log.debug.

Also: return significance image, so that callers can get it directly on the line where it's computed if they like.
If the prefer, they can also access it from the maps data member later.